lab4/lab4_task2.css

```diff
@@ -0,0 +1,119 @@
+        input#whiteBg:checked ~ article {               
+            background-color: #fff;
+            color: #000;
+        }
+        input#bigFont:checked ~ article {               
+            font-size: larger;
+        }
+        input#smallFont:checked ~ article {               
+            font-size: smaller;
+        }
+        header, footer {
+            margin: 10px;
+            padding: 10px;
+            min-width: 640px;
+            width: 100%;
+            border: 1px solid #eee; 
+        }
+        nav {            
+            padding: 10px;
+            margin: 10px;
+            background-color: #CDFCA9;
+            /*width: 100%;*/
+            border: 1px solid #eee;
+            float: left;
+            min-width: 640px;
+            width: 100%;
+            position: -webkit-sticky;
+            position: sticky;
+            top: 0;                                  
+        }
+        nav h1 {
+            display: block;
+        }
+        nav > ul {
+            list-style-type: none;
+            padding: 0px;            
+        }
+        nav li { 
+            float: left; 
+        } 
+        nav li a { 
+            display: block; 
+            width: 80px; 
+            background-color:#E8B15B; 
+        } 
+        main {            
+            width: 100%;
+            background: #0094ff;
+            padding: 10px;
+            margin: 10px;
+            float: left;
+            /*position: relative;*/
+        }
+        article {
+            /*min-width: 200px;
+            max-width: 300px;*/
+            width: 99%;
+            margin: auto;
+            padding: 5px;            
+            float: left;
+            background-color: #E8B15B;
+            column-count: 3;
+            column-gap: 10px;
+            border: dotted 1px gray;
+        }
+        article > p::first-letter {
+            font-family: sans-serif;
+            font-size: xx-large;
+        }
+        article > p {
+            text-align: justify;
+            hyphens: auto;
+        }
+        article .colSpan {
+            -webkit-column-span: all;
+            column-span: all;
+            margin: 0px;
+        }
+        figure {
+            padding: 0px;
+            margin: auto;
+            column-span: all;
+            margin: auto;
+            width: 50%;
+            text-align: center;
+            font-family: 'Anton', sans-serif; 
+        }
+        aside {
+            /*height: 200px;
+            width: 200px;*/
+            padding:10px;
+            background-color: #C1D652;
+            position: fixed;
+            right: 10px;
+            bottom: 10px;            
+        }
+        aside > ul {
+            list-style-type: none;
+            padding: 0px;
+        }        
+        footer {
+            clear: both;
+        }
+        @import url('https://fonts.googleapis.com/css?family=Arvo');
+        @import url('https://fonts.googleapis.com/css?family=Anton');
```

Comment:
The main idea of this task is to make it look as close to the real paper newspaper as possible. Currently it's far from that.
A screenshot of how it could look:
![newspaper](https://user-images.githubusercontent.com/16776066/31585194-875d236e-b1c6-11e7-90e7-04ed42543b30.png)